### PR TITLE
Add Raft to Archipelago

### PR DESCRIPTION
**Pull request by SunnyBat**
Add Raft to Archipelago

- Locations include notes, blueprints, Research Table, and specific quests
- Items include craftable non-decoration items and frequencies
- Optional resource packs and duplicate items included to even out amount of locations vs items
- Tests are all passing (no non-autogenerated tests added at this time)
- No custom tracker has been added at this time

### CODE REVIEW

**Comment by Ijwu** [comment]
Thanks for this. We're looking everything over and waiting for the chance to play a beta game. Let us know when multiple players may play in the same session, that would make things very interesting!

File: worlds/raft/Rules.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
+from ..generic.Rules import set_rule
+from .Locations import location_table
+from .Regions import regionMap
+from ..AutoWorld import LogicMixin
+
+class RaftLogic(LogicMixin):
+    def paddleboard_mode_enabled(self, player):
```
Comment (JarnoWesthof) [inline worlds/raft/Rules.py:7]: logic mixins are all loaded together on the collection state object, therefor its highly recommended to make custom logicMixin methods have a prefix of the game like _raft_

File: WebHostLib/static/assets/tutorial/tutorials.json [hunk 2]
```diff
@@ -247,6 +247,26 @@
         ]
       }
     ]
+  },,
```
Comment (JarnoWesthof) [inline WebHostLib/static/assets/tutorial/tutorials.json:250]: double ,

**Review by JarnoWesthof — CHANGES_REQUESTED** [review]

**Comment by Berserker66** [comment]
Thank you!

**Pull request merged**